packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/plugin-ledger-connector-iroha.ts

```diff
@@ -0,0 +1,684 @@
+            accountId: req.params[0],
+            publicKey: req.params[1],
+          });
+          return { transactionReceipt: response };
+        } catch (err) {
+          throw new Error(err);
+        }
+      }
+      case IrohaQuery.GetRoles: {
+        try {
+          const response = await queries.getRoles(queryOptions);
+          return { transactionReceipt: response };
+        } catch (err) {
+          throw new Error(err);
+        }
+      }
+      case IrohaCommand.CreateRole: {
+        try {
+          const response = await commands.createRole(commandOptions, {
+            roleName: req.params[0],
+            permissionsList: req.params[1],
+          });
+          return { transactionReceipt: response };
+        } catch (err) {
+          throw new Error(err);
+        }
+      }
+      case IrohaCommand.AppendRole: {
+        try {
+          const response = await commands.appendRole(commandOptions, {
+            accountId: req.params[0],
+            roleName: req.params[1],
+          });
+          return { transactionReceipt: response };
+        } catch (err) {
+          throw new Error(err);
+        }
+      }
+      case IrohaCommand.DetachRole: {
+        try {
+          const response = await commands.detachRole(commandOptions, {
+            accountId: req.params[0],
+            roleName: req.params[1],
+          });
+          return { transactionReceipt: response };
+        } catch (err) {
+          throw new Error(err);
+        }
+      }
+      case IrohaQuery.GetRolePermissions: {
+        try {
+          const response = await queries.getRolePermissions(queryOptions, {
+            roleId: req.params[0],
+          });
+          return { transactionReceipt: response };
+        } catch (err) {
+          throw new Error(err);
+        }
+      }
+      case IrohaCommand.GrantPermission: {
+        try {
+          type permission = keyof GrantablePermissionMap;
+          const response = await commands.grantPermission(commandOptions, {
+            accountId: req.params[0],
+            permission: GrantablePermission[req.params[1] as permission],
+          });
+          return { transactionReceipt: response };
+        } catch (err) {
+          throw new Error(err);
+        }
+      }
+      case IrohaCommand.RevokePermission: {
+        try {
+          type permission = keyof GrantablePermissionMap;
+          const response = await commands.revokePermission(commandOptions, {
+            accountId: req.params[0],
+            permission: GrantablePermission[req.params[1] as permission],
+          });
+          return { transactionReceipt: response };
+        } catch (err) {
+          throw new Error(err);
+        }
+      }
+      case IrohaCommand.SetSettingValue: {
+        throw new Http405NotAllowedError("SetSettingValue is not supported.");
+      }
+      case IrohaQuery.GetTransactions: {
+        try {
+          // {txHashesList!: Array<string>;}
```

Comment:
@hxlaf Remove commented code like this unless it's for a specific reason

Reply:
Removed! Thank you!